sandboxie-plus/Sandboxie
# Icons and 3 new toolbar actions

Feature:
* Provide 3 additional icons for actions and enable these actions in the toolbar

QA suggestions:
* Ensure these icons and transparency work on different OS than Win10x64
* Ensure they work in dark/light modes
* Ensure toggle states for actions work in toolbar and menu
* Ensure load/save of toolbar state works for these actions

![image](https://github.com/sandboxie-plus/Sandboxie/assets/28550406/f10cba6d-46f0-4e38-b11e-f014980468cf)
